pkg/logger/logger.go

```diff
@@ -271,7 +272,19 @@ func setLogConf(logConfig string) {
 		_ = ioutil.WriteFile(path, []byte(logConfigContent), os.ModePerm)
```

Comment:
can we use os.WriteFile

Reply:
I see many packages using ioutil.WriteFile(alerts from idle: ioutil.WriteFile is deprecated: As of Go 1.16, this function simply calls os.WriteFile), i will change here, if necessary, I will make another pr to fix in other packages

Reply:
Just fix this one is fine. You can create another PR to fix code in other files